src/components/tile-details-panel/tile-validator.tsx

```diff
@@ -1,41 +1,32 @@
 import type { Tile3D } from "@loaders.gl/tiles";
 import { useEffect, useState } from "react";
 import styled from "styled-components";
-import { Checkbox } from "../";
-import { color_canvas_primary_inverted } from "../../constants/colors";
-import { SelectionState } from "../../types";
```

Comment:
Check removed imports. If they are not used in other places, remove them as well